docs/index.md

```diff
@@ -65,76 +69,76 @@ POST {json file} <url>/insert/
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2012-03-20", "sample_type": "V", "result": "13.6"}, 
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2012-02-21", "sample_type": "V", "result": "15"}, 
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2011-08-19", "sample_type": "V", "result": "5"}] 
 }
 ```
 
 ### Response
 
 #### Body
 
-| Field            | Format       | Constraints                                                                     | Notes/Questions                                                                                  |
-|------------------|--------------|---------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
-| version          | Text         | "0.3.0"                                                                         | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
-| timestamp        | TimeDate     | yyyy-mm-ddThh:mm:ss.sss-hh:mm                                                   | Current Timestamp. RFC 3339 (ISO 8601 derivative).                                               |
-| visit_date       | Date         | yyyy-mm-dd                                                                      | Effective date of score                                                                          |
-| alliance_id      | Text         | AlphaNumeric                                                                    | Return value provided by Feed                                                                    |
-| risk_score       | AlphaNumeric | 9.99                                                                            | Currently expecting numeric, future may be phrased by stating an odds ratio with the risk score. |
-| risk_score_notes | Text         | Additional notes (referral, remediation funds, etc to be returned to provider ) |                                                                                                  |
+| Field            | Format       | Constraints                       | Notes/Questions                                                                                  |
+|------------------|--------------|-----------------------------------|--------------------------------------------------------------------------------------------------|
+| version          | Text         | "0.3.0"                           | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
+| timestamp        | TimeDate     | yyyy-mm-dd hh:mm:ss.sss-hh:mm     | Current Timestamp (RFC 3339 complient date).                                                     |
+| visit_date       | Date         | yyyy-mm-dd                        | Effective date of score                                                                          |
```

Comment:
The reason I originally suggested it was because of both past and future visit dates. But:

1) past visit dates will, I believe, only be used internally for validation and that will probably happen by calling `predict.py` directly, no reason to do it over the API. if an external provider submits a past visit date, it seems like we'd want to handle that as an error.

2) future visit dates are plausible for providers, buta future visit date won't affect the returned prediction because we don't have future models trained on future data to use for the prediction, just the current one.

So not having a visit date makes things simpler. The only reason I can see for having one in the public API is to collect data on when patients are visiting, but we have that in the `visit` array. Sorry to go back on my original suggestion.